app/routes/queries/full-scan/index.tsx

```diff
@@ -260,10 +257,15 @@ const Index = () => {
                         Filters let you limit what type of item you're wanting
                         to search for. Keep in mind, that a low Scan Hours or
                         high Sale Amount value will limit your results.
+                        <span className={`italic invisible sm:visible`}>
+                          Spicy side note, this is a multi-select filter. Hold
```

Comment:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/17516896/190887157-96b76c73-6cf9-467b-98fb-5fda87c3ecf8.png">
There needs to be spaces inbetween the `.` and `Spicy`